go.mod

```diff
@@ -327,6 +327,7 @@ require (
 
 require (
 	github.com/coreos/go-oidc/v3 v3.9.0
+	github.com/dghubble/oauth1 v0.7.2
```

Comment:
> To implement "Login with X", use the [gologin](https://github.com/dghubble/gologin) packages which provide login handlers for OAuth1 and OAuth2 providers.
>
> To call the Twitter, Digits, or Tumblr OAuth1 APIs, use the higher level Go API clients.

Did you check that note in the repo's readme?

Reply:
The usage later looks valid IMO.

Reply:
I think this should be fine!